store/tikv/txn.go

```diff
@@ -359,6 +360,7 @@ func (txn *tikvTxn) Rollback() error {
 	// Clean up pessimistic lock.
 	if txn.IsPessimistic() && txn.committer != nil {
 		err := txn.rollbackPessimisticLocks()
+		txn.committer.ttlManager.close()
```

Comment:
maybe take care one special case:

1. we meet `ErrEntryTooLarge` in txn Commit at:

https://github.com/pingcap/tidb/blob/0b29c2b47770293a904179ae8bf317d925a4fea2/store/tikv/txn.go#L305

2. then defer in Commit will close txn and mark `txn.valid = false`

https://github.com/pingcap/tidb/blob/0b29c2b47770293a904179ae8bf317d925a4fea2/store/tikv/txn.go#L274

3. so txn is invalid and client maybe send rollback again, but at Rollback method will return at line 359

ttl goroutine will have no chance to exit